cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -264,3 +266,21 @@ def test_reports_delete_saved_report(appliance, request):
     view.configuration.item_select(
         item='Delete selected Saved Reports', handle_alert=True)
     assert not report.exists
+
+
+def test_reports_create_schedule_for_custom_report(
```

Comment:
Required:
Better to called it `test_custom_schedule_report_crud` what you say? and please delete what you created.

Reply:
Okay, we can name it that. I delete report at the end of the test, which automatically deletes the schedule.